app/views/pages/moving-guide/claims.html.erb

```diff
@@ -0,0 +1,198 @@
+        </div>
+      </div>
+    </div>
+
+    <div class="what-to-expect-section" id="full-replacement-value">
+      <h4>What does "Full Replacement" Value" mean?</h4>
+
+      <p>When you are moving with the military, your items are insured at “Full Replacement Value” at no additional cost to the service family. Essentially, this means that if an item is lost or destroyed during the move, the carrier should pay to replace it with a like item. The replacement could be new, used, or if the item is damaged repaired to fulfill the “Full Replacement Value” policy. The carrier will still require proof of the item's value and quality and will likely want evidence of the cost to replace it.</p>
+    </div>
+
+    <div class="what-to-expect-section" id="report">
+      <h4>Loss & Damage Report</h4>
+
+      <p>A Loss/Damage Report allows you to notify the moving company of your intent to file a claim for missing or damaged items. The big advantage of filing the Loss/Damage report is that it extends the amount of time you can file claims and still receive the full replacement value. <strong>Remember… a Loss/Damage Report is NOT a claim, you must still file a itemized claim to receive reimbursement!</strong></p>
+
+      <p><strong>Damage found on Pack-Up/Delivery Day:</strong> If you notice any lost or damaged items on delivery day, you should ask the driver to complete and sign a DD1840 form. (The driver should have copies of this form on hand, but if you want to be super prepared, you can print a few copies yourself.) Of course you can always report these items after the fact, but the advantage of doing it on moving day is that both you and your transportation service provider (TSP) are agreeing that damage occured up front. That can lead to less disputes later on in the claims process and potentially higher settlements. If you and the driver complete a paper DD1840 form, a copy of this document must be uploaded to the <%= link_to 'Defense Personal Property System (DPS)', 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %> within 30 days of your delivery date.</p>
+
+      <p><%= link_to 'For help submitting a Loss & Damage Report, follow this step-by-step tutorial.', page_path('tutorials') %></p>
+
+      <p>
+        <strong>Important Dates:</strong>
+        <br>
+        Submitting a Loss & Damage Report is optional, but more time and flexibility.
+      </p>
+
+      <table>
+        <thead>
+          <tr>
+            <th colspan="2">Important Dates WITH Loss/Damage Report</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th scope="row">Delivery Day (Optional)</th>
+            <td>For any lost or damaged items discovered on delivery day, request that your moving driver complete and sign a DD1840 form. Having mutual agreement on major items, can lead to less disputes when you settle the reimbursement value later on.</td>
```

Comment:
Remove the comma from "Having mutual agreement on major items, can lead to less disputes when you settle the reimbursement value later on."

Change "less" to "fewer".